lib/generators/react/install_generator.rb

```diff
@@ -60,8 +60,8 @@ def webpacker?
 
       def javascript_dir
         if webpacker?
-          Webpacker::Configuration.source_path
-            .join(Webpacker::Configuration.entry_path)
+          webpack_configuration.source_path
+            .join(webpack_configuration.entry_path)
```

Comment:
The new singleton implementation of `Webpacker::Configuration` no longer has `entry_path`. Instead, `Webpacker::Configuration.source_path.join(Webpacker::Configuration.entry_path)` becomes `Webpacker.config.source_entry_path`

Reply:
I have suggested these changes in issue #778 .Please have a look into it. I was trying to install react with rails and failed to run react installer command. I look into gem and found that method calling is not proper. 

I fix this with minor changes in `lib/generators/react/install_generator.rb` file.

`source_path` is a instance method and was being called by class directly. Also `entry_path` was changed to `source_entry_path` in webpacker config. I have suggested these changes in #778 

The main purpose for issue was #778 was to elevate exact issue and its solution along with it, as there was no fix until that time.

Reply:
Using webpacker 3.0.1 Webpacker.config.source_entry_path worked for me 👍